src/index.tsx

```diff
@@ -44,42 +44,71 @@ function SquircleBackground({
   strokeColor = '#000',
   strokeWidth = 0,
 }: SquircleParams) {
-  const [squircleSize, setSquircleSize] =
+  return (
+    <Rect style={StyleSheet.absoluteFill}>
+      {({ width, height }) => {
+        const squirclePath = getSvgPath({
+          width: width,
+          height: height,
+          cornerSmoothing,
+          cornerRadius,
+          topLeftCornerRadius,
+          topRightCornerRadius,
+          bottomRightCornerRadius,
+          bottomLeftCornerRadius,
+        })
+
+        if (strokeWidth > 0) {
+          // Since SVG doesn't support inner stroke, we double the stroke width
+          // and remove the outer half with clipPath
+          return (
+            <Svg>
+              <Defs>
+                <ClipPath id="clip">
+                  <Path d={squirclePath} />
+                </ClipPath>
+              </Defs>
+              <Path
+                clipPath="url(#clip)"
+                d={squirclePath}
+                fill={fillColor}
+                stroke={strokeColor}
+                strokeWidth={strokeWidth * 2}
+              />
```

Comment:
Not sure how this'll work, will checkout and play with it locally.

Reply:
Sure take your time. Let me know if you have any questions.

Reply:
I was playing with Figma and thought this might help. `squirclePath` is the blue line, we double the stroke width and use the same path as a clipping path to remove the outer stroke.
<img width="403" alt="Screen Shot 2022-03-21 at 3 09 30 PM" src="https://user-images.githubusercontent.com/57227217/159224588-7f1d1d66-fa7f-4757-a81b-73fe8fcb9bf2.png">